api/src/pcapi/routes/saml/educonnect.py

```diff
@@ -64,13 +64,16 @@ def on_educonnect_authentication_response() -> Response:  # pylint: disable=too-
         error_query_param = {"code": "UserTypeNotStudent"}
         return redirect(ERROR_PAGE_URL + urlencode(error_query_param), code=302)
 
-    except educonnect_exceptions.EduconnectAuthenticationException:
-        logger.warning("Educonnect authentication Error")
+    except Exception as exc:  # pylint: disable=broad-except
+        logger.error("Error after educonnect authentication: %s", exc)
```

Comment:
```suggestion
        logger.exception("Error after educonnect authentication: %s", exc)
```

Ça permettra d'avoir tout le traceback dans Sentry.